### PR TITLE
Minor. Update api doc about source update regarding auto_response_triggers and signatures. RD-22688 RD-22689 RD-22690 RD-22691

### PR DESCRIPTION
**Pull request by onima**
## Jira Ticket :  
* https://jira.ringcentral.com/browse/RD-22688
* https://jira.ringcentral.com/browse/RD-22689
* https://jira.ringcentral.com/browse/RD-22690
* https://jira.ringcentral.com/browse/RD-22691

We update API documentation for PUT /1.0/content_sources endpoint now that we give possibility to update with our API auto_response_triggers logic, signatures and content_languages.

### CODE REVIEW

File: specs/engage-digital_openapi3.yaml [hunk 1]
```diff
@@ -7900,6 +7958,42 @@ components:
         enable_web:
           description: Only on Engage Messaging sources. Boolean describing if the source is enabled on the web or not.
           type: boolean
+        auto_response_triggers:
```
Comment (onima) [inline specs/engage-digital_openapi3.yaml:7961]: response_api_update_source

File: specs/engage-digital_openapi3.yaml [hunk 2]
```diff
@@ -715,6 +715,56 @@ paths:
             items:
               type: string
             type: array
+        - description: Signatures
```
Comment (onima) [inline specs/engage-digital_openapi3.yaml:718]: https://user-images.githubusercontent.com/8244522/187923496-a05d0d1d-0b44-4e0e-bc3a-90a8a8e4d8d1.mov

**Comment by ylecuyer** [comment]
Could you make it so that it says it is an array of object with body/language:

Screenshot_2022-09-02_15-35-59


Screenshot_2022-09-02_15-36-33

**Comment by ylecuyer** [comment]
Same thing

Screenshot_2022-09-02_15-40-52


should be an array of objects language/created_from_trigger/body

File: specs/engage-digital_openapi3.yaml [hunk 3]
```diff
@@ -884,6 +884,48 @@ paths:
...
             type: array
+        - description: Signatures
+          in: query
+          name: signatures
+          required: false
+          schema:
+            type: array
+            items:
+              type: object
+            example: [{ body: 'Hello World', language: 'en' }]
```
Comment (onima) [inline specs/engage-digital_openapi3.yaml:895]: Fixed : 

Screenshot 2022-09-05 at 13 33 38

I try to display it on the left section but it's not supported yet -> There is an opening issue for it : https://github.com/swagger-api/swagger-ui/issues/4581

**Comment by ylecuyer** [comment]
LGTM

**Pull request merged**